packages/strapi-plugin-email/controllers/Email.js

```diff
@@ -45,17 +40,7 @@ module.exports = {
   },
 
   getEnvironments: async ctx => {
-    const environments = _.map(
-      _.keys(strapi.config.environments),
-      environment => {
-        return {
-          name: environment,
-          active: strapi.config.environment === environment,
-        };
-      }
-    );
-
-    ctx.send({ environments });
+    ctx.send({ environments: [] });
```

Comment:
Is this still relevant to return a static empty value? Who's using this?

Reply:
Nope was a temp patch :) I'm removing this completely because we are moving the email config to files.